src/NLPModels.jl

```diff
@@ -172,6 +173,19 @@ function objgrad!(nlp, x, g)
   return f, g
 end
 
+"""`(rows,cols) = jac_structure(nlp)`
+
+Returns the structure of the constraint's Jacobian in sparse coordinate format.
+"""
+jac_structure(:: AbstractNLPModel) = throw(NotImplementedError("jac_structure"))
+
+"""`(rows,cols,vals) = jac_coord!(nlp, x, rows, cols, vals)`
+
+Evaluate \$\\nabla c(x)\$, the constraint's Jacobian at `x` in sparse coordinate format,
```

Comment:
How does the LaTeX show in the console? Is it readable?

Reply:
Here, but `\nabla c(x)` is colored differently, with a different color that `x`, `vals`, etc.
```
  (rows,cols,vals) = jac_coord!(nlp, x, rows, cols, vals)

  Evaluate \nabla c(x), the constraint's Jacobian at x in sparse coordinate format, rewriting vals.
  rows and cols are not rewritten.
```